operator/pkg/controller-runtime/cell.go

```diff
@@ -51,10 +51,10 @@ func newScheme() (*runtime.Scheme, error) {
 type managerParams struct {
 	cell.In
 
-	Logger      logrus.FieldLogger
-	Lifecycle   cell.Lifecycle
-	JobRegistry job.Registry
-	Scope       cell.Scope
+	Logger    logrus.FieldLogger
+	Lifecycle cell.Lifecycle
+	JobGroup  job.Group
+	Scope     cell.Health
```

Comment:
nit: there are some leftovers of `Scope cell.Health` that probably should be renamed to `Health`.